dashboard/app/views/pd/application/teacher_application_mailer/needs_admin_approval.html.haml

```diff
@@ -6,40 +6,31 @@
   = @application.effective_regional_partner_name
   has begun reviewing your application. Based on that review,
   = @application.effective_regional_partner_name
-  has determined that your application requires approval from you school's
+  has determined that your application requires approval from your school's
   administrator/leadership for you to be considered for acceptance into the
   Program.
 
 %p
-  Please note that
-  = @application.effective_regional_partner_name + "'s"
-  requires approval from your school's administration/leadership for you to be considered for acceptance into the
-  Program. The school administrator/leader that you indicated in your application received an email with the subject
+  The school administrator/leader that you indicated in your application received an email with the subject
   "Action Needed:
   = @application.first_name + ' ' + @application.last_name
   has applied to
   = @application.effective_regional_partner_name + "'s"
   Professional Learning Program!" with a link to the approval form. To prevent delay, we encourage you to also contact
   them directly to make them aware of the approval requirement. If they can not find the email you can also give them
   this unique link to the
-  = link_to 'Administrator/School Leader approval form', @application.principal_approval_url
-  for your application.
+  = link_to 'Administrator/School Leader approval form for your application.', @application.principal_approval_url
```

Comment:
Nit: would you mind structuring this line the same way it's structured [here](https://github.com/code-dot-org/code-dot-org/pull/54998/files#diff-4f2063d64710477fdde082523d413f982a0bc2b9e568ec8f833d1e2355fd2dc0R40) so that the period is separated out from being hyperlinked and follows the same pattern below